docs/_theme/assets/style.css

```diff
@@ -11,6 +11,15 @@
   padding-bottom: 3rem;
 }
 
+.caret-right {
+  color: rgba(0, 0, 0, 0.6);
+  -moz-transform: scaleX(-1);
+  -o-transform: scaleX(-1);
```

Comment:
Opera moved to Blink, so -webkit- covers that. Unless you're trying to catch the 2 people using a really old version of Opera?